bugbug/models/component.py

```diff
@@ -133,6 +133,10 @@ def __init__(self, lemmatization=False):
     def filter_component(self, product, component):
         if product == "Fenix":
             return "Fenix"
+
+        if product == "GeckoView":
+            return "GeckoView::General"
```

Comment:
It might be better to combine all of them together as a single class, the model should perform better this way

Reply:
Yep, I was planning on doing so after training the model once! I'll be sure to compare the results as well. My only concern with combining them into a single class is if the bug is not classified by the Fenix/GeckoView-specific model, it will default to the component model classification (which would be our combined class). Should the combined class simply be `Fenix::General`?

Reply:
I see your point.
Yes, I think it'd be ideal to fallback on `Fenix::General` when the specific model isn't sure.